src/cloud-api-adaptor/ibmcloud-powervs/README.md

```diff
@@ -30,27 +30,33 @@ pvsadm image qcow2ova --prep-template-default > image-prep.template
 
 Add the following snippet to `image-prep.template`
 ```
-yum install -y gcc gcc-c++ git make
+yum install -y gcc gcc-c++ git make wget perl
+wget https://github.com/mikefarah/yq/releases/download/v4.42.1/yq_linux_ppc64le
+chmod +x yq_linux_ppc64le && mv yq_linux_ppc64le /usr/local/bin/yq
 git clone -b main https://github.com/kata-containers/kata-containers.git
 git clone https://github.com/confidential-containers/cloud-api-adaptor.git
 cd cloud-api-adaptor/ibmcloud-powervs/image
-make build
+AGENT_POLICY=no SEALED_SECRET=no make build
```

Comment:
Is there are reason that you are disabling AGENT_POLICY? This is a feature we are using in other cloud providers. The same will likely be true for SEALED_SECRET soon (though it isn't an option at the moment)

Reply:
We haven't given a try at the moment with these features being enabled, will revert the change for now, once we test these features we can have some idea on it and will raise additional PR for these changes if required.